azure-kusto-data/azure/kusto/data/request.py

```diff
@@ -262,6 +266,7 @@ def __init__(self, kcsb):
 
         self._mgmt_endpoint = "{0}/v1/rest/mgmt".format(kusto_cluster)
         self._query_endpoint = "{0}/v2/rest/query".format(kusto_cluster)
+        self._streaming_ingest_endpoint = "{0}/v1/rest/ingest/".format(kusto_cluster).replace("ingest-", "")

```

Comment:
should probably inline the replace (shorter string)
```suggestion
        self._streaming_ingest_endpoint = "{0}/v1/rest/ingest/".format(kusto_cluster.replace("ingest-", ""))
```